doc/sign-package.md

```diff
@@ -79,43 +81,39 @@ Included is a sample JSON configuration where the elements will be described nex
-UnhandledSyscallEnosys | This option would prevent the termination of a program using myst_panic when an unimplemented syscall is encountered in the mystikos kernel. The default value is `false`, which implies that we terminate on unhandled syscalls by default. If `true`, it will cause the syscall to return ENOSYS error.
+Settings | Type | Description
+-|-|-
+(Deprecated) UserMemSize | `int \| string` | Deprecated, use MemorySize instead
+MemorySize | `int \| string` | Amount of memory your application needs to run. Try not to make this just a very large number as the larger this number needs to be the slower load time will be. Value can be bytes (just a number), Kilobytes (number with k after, for example `"128k"`), or megabytes (number with m after, for example `"512m"`)
+MaxAffinityCPUs | `int` | This setting limits the number of CPUs reported by sched_getaffinity()
+ApplicationPath | `string` | The executable path relative to the root of your appdir. This executable name is also used to determine the final application name once packaged.
+HostApplicationParameters | `boolean \| int` | This parameter specifies if application parameters can be specified on the command line or not. If true, the command line arguments are used instead of the ApplicationParameters
+ApplicationParameters | `[string] \| string` | Enclave defined application parameters if HostApplicationParameters is set to false.
+EnvironmentVariables | `[string] \| string` | Enclave defined environment variables, a string should be a key value pair like `"KEY=VALUE"`
+HostEnvironmentVariables | `[string] \| string` | A string denoting a single variable , or a list of environment variables that can be imported from the insecure host
+CurrentWorkingDirectory | `string` | The default working directory for the application
+Hostname | `string` | The default hostname exposed to application
+ForkMode | `string` | Specify the mode used for the experimental pseudo fork feature. Refer to [doc/design/fork.md](/doc/design/fork.md) for more details. The default value is `"none"`, which disables the feature.
+Mount | | Set if parameters for informing Mystikos to automatically mount a set of directories or ext2 disk images from the host into the TEE. Refer to [doc/design/mount-config-design.md](/doc/design/mount-config-design.md) for more details. By default no extra mounts are added to the root filesystem.
+UnhandledSyscallEnosys | `boolean \| int` | This option would prevent the termination of a program using myst_panic when an unimplemented syscall is encountered in the mystikos kernel. The default value is `false`, which implies that we terminate on unhandled syscalls by default. If `true`, it will cause the syscall to return ENOSYS error.
```

Comment:
```suggestion
UnhandledSyscallEnosys | `boolean \| int` | This option would prevent the termination of a program using myst_panic when an unimplemented syscall is encountered in the Mystikos kernel. The default value is `false`, which implies that we terminate on unhandled syscalls by default. If `true`, it will cause the syscall to return an ENOSYS error.
```